src/main/java/com/example/gradecalculator/web/model/LoginDTO.java

```diff
@@ -0,0 +1,40 @@
+package com.example.gradecalculator.web.model;
+
+import lombok.Getter;
+
+@Getter
+public class LoginDTO {
```

Comment:
this class is not used anymore, as the custom login implementation was replaced with the built in one.
you should remove this class